src/consensus/dkg.rs

```diff
@@ -260,4 +267,24 @@ impl DkgVoter {
     pub fn push_vote(&mut self, vote: Vote, proof: Proof) {
         self.pending_accumulated_votes.push_front((vote, proof));
     }
+
+    // Return with true only when accumulated quorum valid notifications first time.
+    pub fn add_old_elders_notification(
+        &mut self,
+        participants: &BTreeSet<PublicId>,
+        src_id: &PublicId,
+    ) -> bool {
+        if !participants.contains(src_id) {
+            return false;
+        }
+        if let Some(senders) = self.old_elders_notifications.get_mut(participants) {
+            let _ = senders.insert(*src_id);
+            return senders.len() == threshold_count(participants.len());
+        }
+        let senders: BTreeSet<_> = iter::once(*src_id).collect();
+        let _ = self
+            .old_elders_notifications
+            .insert(participants.clone(), senders);
```

Comment:
This can be made more succinct:

```rust
let senders = self.old_elders_notifications.entry(participants.clone()).or_default();
let _ = senders.insert(*src_id);
senders.len() == threshold_count(participants.len())
```

Reply:
still has to be something like:
```rust
        match self.old_elders_notifications.entry(participants.clone()) {
            Entry::Vacant(entry) => {
                let senders: BTreeSet<_> = iter::once(*src_id).collect();
                let _ = entry.insert(senders);
                false
            }
            Entry::Occupied(entry) => {
                let value = entry.into_mut();
                let _ = value.insert(*src_id);
                value.len() == threshold_count(participants.len())
            }
        }
```
as LruTimeCache has it's own definition of Entry